File: 1534. Count Good Triplets/countGoodTriplets.js
var countGoodTriplets = function (arr, a, b, c) {
    let ans = 0;
    let min = Math.min(a, c)

    //Brute Force
    // for(let i=0 ; i<arr.length-2;i++){

    //     for(let j=i+1; j<arr.length-1; j++){

    //         for(let k=j+1 ; k<arr.length; k++){

    //             if(
    //                 Math.abs(arr[i] - arr[j]) <= a &&
    //                 Math.abs(arr[j] - arr[k]) <= b &&
    //                 Math.abs(arr[i] - arr[k]) <= c
    //             ){
    //                 ans++
    //             }
    //         }
    //     }
    // }

    // return ans

    for (let i = 0; i < arr.length - 2; i++) {

        for (let j = i + 1; j < arr.length - 1; j++) {

            if (Math.abs(arr[i] - arr[j]) > a) continue;
            for (let k = j + 1; k < arr.length; k++) {
                if (
                    Math.abs(arr[j] - arr[k]) <= b &&
                    Math.abs(arr[i] - arr[k]) <= c
                ) {
                    ans++
                }
            }
        }
    }

    return ans

};

File: 1534. Count Good Triplets/countGoodTriplets.cpp
using namespace std;
#include <iostream>

class Solution {
public:
    int countGoodTriplets(vector<int>& arr, int a, int b, int c) {
        int ans = 0;
        int sz = arr.size();

        for(int i=0 ; i<sz-2; i++){

            for(int j=i+1; j<sz-1; j++){

                if(abs(arr[i] - arr[j]) > a) continue;

                for(int k=j+1; k<sz; k++){

                    if(abs(arr[j] - arr[k]) <= b && abs(arr[i] - arr[k]) <= c){
                        ans++;
                    }
                }
            }
        }

        return ans;
    }
};